modules/meetup/queries/numbers.api.ts

```diff
@@ -0,0 +1,76 @@
+import { gql } from 'graphql-request';
+import { client, LYONJS_MEETUP_ID } from '../api';
+
+const query = gql`
+  query meetup($id: ID!) {
+    group(id: $id) {
+      pastEvents(input: { first: 5000 }) {
+        edges {
+          node {
+            going
+          }
+        }
+      }
+    }
+  }
+`;
+
+type Node = {
+  node: {
+    going: number;
+  };
+};
+
+type PastEvents = {
+  pastEvents: {
+    edges: Node[];
+  };
+};
+
+const getAge = () => {
+  var today = new Date();
+  var birthDate = new Date('2011-10-25');
+  var age = today.getFullYear() - birthDate.getFullYear();
+  var m = today.getMonth() - birthDate.getMonth();
+  if (m < 0 || (m === 0 && today.getDate() < birthDate.getDate())) {
+    age--;
+  }
+  return age;
+};
```

Comment:
I would add explicite return type to improve readibility and use moderne constant definition 😉 
```suggestion
const getAge = (): number => {
  const today = new Date();
  const birthDate = new Date('2011-10-25');
  let age = today.getFullYear() - birthDate.getFullYear();
  const m = today.getMonth() - birthDate.getMonth();
  if (m < 0 || (m === 0 && today.getDate() < birthDate.getDate())) {
    age--;
  }
  return age;
};
```